helm-chart/templates/cvat_kvrocks/statefulset.yml

```diff
@@ -59,6 +59,25 @@ spec:
           {{- with .Values.cvat.kvrocks.additionalEnv }}
           {{- toYaml . | nindent 10 }}
           {{- end }}
+          #https://github.com/apache/kvrocks/blob/unstable/Dockerfile
+          {{- if .Values.cvat.kvrocks.readinessProbe.enabled }}
+          readinessProbe:
+            exec:
+              command:
+              - /bin/sh
+              - -c
+              - ./bin/redis-cli -p 6666 PING | grep -E '(PONG|NOAUTH)' || exit 1
```

Comment:
I don't think `|| exit 1` does anything useful here. It just overwrites the previous error code.